pkg/checkmarx/checkmarx.go

```diff
@@ -525,14 +525,14 @@ func (sys *SystemInstance) UpdateProjectConfiguration(projectID int, presetID in
 }
 
 // ScanProject triggers a scan on the project addressed by projectID
-func (sys *SystemInstance) ScanProject(projectID int, isIncremental, isPublic, forceScan bool) (bool, Scan) {
+func (sys *SystemInstance) ScanProject(projectID int, isIncremental, isPublic, forceScan bool, filterPattern  string) (bool, Scan) {
 	scan := Scan{}
 	jsonData := map[string]interface{}{
 		"projectId":     projectID,
 		"isIncremental": isIncremental,
 		"isPublic":      isPublic,
 		"forceScan":     forceScan,
-		"comment":       "Scan From Golang Script",
+		"comment":       fmt.Sprintf("Scan From Golang Script using FILTER_PATTERN=%s", filterPattern),
```

Comment:
The commit says that this was added for debugging. Does it need to be removed again?